feg/gateway/services/session_proxy/credit_control/gy/config.go

```diff
@@ -27,32 +27,35 @@ import (
 
 // OCS Environment Variables
 const (
-	OCSAddrEnv              = "OCS_ADDR"
-	GyNetworkEnv            = "GY_NETWORK"
-	GyDiamHostEnv           = "GY_DIAM_HOST"
-	GyDiamRealmEnv          = "GY_DIAM_REALM"
-	GyDiamProductEnv        = "GY_DIAM_PRODUCT"
-	GyInitMethodEnv         = "GY_INIT_METHOD"
-	GyLocalAddr             = "GY_LOCAL_ADDR"
-	OCSHostEnv              = "OCS_HOST"
-	OCSRealmEnv             = "OCS_REALM"
-	OCSApnOverwriteEnv      = "OCS_APN_OVERWRITE"
-	OCSServiceIdentifierEnv = "OCS_SERVICE_IDENTIFIER_OVERWRITE"
-	DisableDestHostEnv      = "DISABLE_DEST_HOST"
-	OverwriteDestHostEnv    = "GY_OVERWRITE_DEST_HOST"
-	UseGyForAuthOnlyEnv     = "USE_GY_FOR_AUTH_ONLY"
-	GySupportedVendorIDsEnv = "GY_SUPPORTED_VENDOR_IDS"
-	GyServiceContextIdEnv   = "GY_SERVICE_CONTEXT_ID"
-
-	GyInitMethodFlag         = "gy_init_method"
-	OCSApnOverwriteFlag      = "ocs_apn_overwrite"
-	OCSServiceIdentifierFlag = "ocs_service_identifier_overwrite"
+	OCSAddrEnv                         = "OCS_ADDR"
+	GyNetworkEnv                       = "GY_NETWORK"
+	GyDiamHostEnv                      = "GY_DIAM_HOST"
+	GyDiamRealmEnv                     = "GY_DIAM_REALM"
+	GyDiamProductEnv                   = "GY_DIAM_PRODUCT"
+	GyInitMethodEnv                    = "GY_INIT_METHOD"
+	GyLocalAddr                        = "GY_LOCAL_ADDR"
+	OCSHostEnv                         = "OCS_HOST"
+	OCSRealmEnv                        = "OCS_REALM"
+	OCSApnOverwriteEnv                 = "OCS_APN_OVERWRITE"
+	OCSServiceIdentifierEnv            = "OCS_SERVICE_IDENTIFIER_OVERWRITE"
+	DisableDestHostEnv                 = "DISABLE_DEST_HOST"
+	OverwriteDestHostEnv               = "GY_OVERWRITE_DEST_HOST"
+	UseGyForAuthOnlyEnv                = "USE_GY_FOR_AUTH_ONLY"
+	GySupportedVendorIDsEnv            = "GY_SUPPORTED_VENDOR_IDS"
+	GyServiceContextIdEnv              = "GY_SERVICE_CONTEXT_ID"
+	DisableRequestedGrantedUnitsAVPEnv = "DISABLE_REQUESTED_SERVICE_UNIT_AVP"
+
+	GyInitMethodFlag                    = "gy_init_method"
+	OCSApnOverwriteFlag                 = "ocs_apn_overwrite"
+	OCSServiceIdentifierFlag            = "ocs_service_identifier_overwrite"
+	DisableRequestedGrantedUnitsAVPFlag = "disable_requested_service_unit_AVP"
 )
 
 var (
 	_ = flag.String(GyInitMethodFlag, "", "Gy init method (per_key|per_session)")
 	_ = flag.String(OCSApnOverwriteFlag, "", "OCS APN to use instead of request's APN")
 	_ = flag.String(OCSServiceIdentifierFlag, "", "OCS ServiceIdentifier to use in Gy requests")
+	_ = flag.String(DisableRequestedGrantedUnitsAVPFlag, "", "Disable Requested-Service-Unit AVP (437)")
```

Comment:
I'd make it Bool here.

avp437Flag = flag.Bool(DisableRequestedGrantedUnitsAVPFlag, false, "Disable Requested-Service-Unit AVP (437)")